Example/Controllers/SwiftUI/Helper/View+Tracking.swift

```diff
@@ -0,0 +1,127 @@
+import SwiftUI
+import MiniApp
+
+struct PageTrackingModifier: ViewModifier {
```

Comment:
Do we have `RATViewController` still in use ?

Reply:
not anymore I've removed the unnecessary analytics files